laplace/baselaplace.py

```diff
@@ -664,15 +664,15 @@ def _glm_forward_call(
                 "Prediction path invalid. Check the likelihood, pred_type, link_approx combination!"
             )
 
-    def _glm_predictive_samples(
+    def _glm_functional_samples(
```

Comment:
I think for UX (or DX -- developer experience) purposes, it's better as an instance method. This way, it's coherent with `_nn_functional_samples` since it can't be made static.

PS. Seems like `@staticmethod` is not recommended everywhere I looked.[^1][^2]

[^1]: https://www.seporaitis.net/posts/2020/05/05/python-staticmethod-usage/
[^2]: https://google.github.io/styleguide/pyguide.html#2174-decision

Reply:
Fair points, thanks!